src/sql/workbench/contrib/jobManagement/browser/alertsView.component.ts

```diff
@@ -215,10 +215,10 @@ export class AlertsViewComponent extends JobManagementView implements OnInit, On
 		let resultIndicatorClass = dataContext.enabled ? 'alertview-alertnameindicatorenabled' :
 			'alertview-alertnameindicatordisabled';
 
-		return '<table class="alertview-alertnametable"><tr class="alertview-alertnamerow">' +
-			'<td nowrap class=' + resultIndicatorClass + '></td>' +
-			'<td nowrap class="alertview-alertnametext">' + dataContext.name + '</td>' +
-			'</tr></table>';
+		return `${'<table class="alertview-alertnametable"><tr class="alertview-alertnamerow">' +
+			'<td nowrap class='}${resultIndicatorClass}></td>` +
+			`<td nowrap class="alertview-alertnametext">${dataContext.name}</td>` +
+			`</tr></table>`;
```

Comment:
for better readability, I'd say we should completely get rid of the single quotes and + symbols, I struggled to tell whether the changes look good. 😄  template string will preserve the line breaks.